packages/appeals-service-api/src/services/back-office-v2/formatters/s78/appeal.js

```diff
@@ -57,8 +58,8 @@ exports.formatter = async (appellantSubmission) => {
 	return {
 		casedata: {
 			submissionId: appellantSubmission.appealId,
-			caseType: 'D',
-			caseProcedure: 'written',
+			caseType: APPEAL_CASE_TYPE.W,
```

Comment:
Almost certainly my ignorance of business logic, but how come the Case Type is W for s78 and D for HAS?

Reply:
Does caseType correspond directly to HAS, S78 etc?

Reply:
D is always HAS, so was just copied from HAS incorrectly

I'm not sure yet if we need to change the case type for S78 based on the answers provided, but W should be S78 written

Reply:
Thanks!